Differentiate robot constants

schineaj23 opened the pull request.

schineaj23 committed:
Different offsets for main and practice chassis

schineaj23 committed:
Fix PathplannerLib dependency

TotallyFarhan committed:
Update dependencies

Co-Authored-By: Andrew Schineller <34293975+schineaj23@users.noreply.github.com>

TotallyFarhan committed:
Correct Main robot offsets

TotallyFarhan committed:
Revise constant names

Ayidana-Aboraah commented on src/main/java/frc/robot/RobotContainer.java:
```diff
@@ -31,9 +31,8 @@
   // The robot's subsystems and commands are defined here...
 
-  private final DriveSubsystem driveSubsystem = new DriveSubsystem();
+  private final DriveSubsystem driveSubsystem = new DriveSubsystem(true);
```
Could make where it is the main robot an enemy for better readability, but this works fine performance-wise, so it's purely stylistic.

Ayidana-Aboraah reviewed (approved):
Everything good, just small nitpicks.

schineaj23 committed:
Use Chassis enum for separate robots instead of boolean

schineaj23 committed:
Rename Chassis to ChassisConfiguration

Realised that "Chassis" is a little unclear, we are differentiating
the configurations/constants between them.
